.github/workflows/main.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+
+on: [pull-request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Lint template files
+      uses: "stefanoeb/eslint-action@master"
+      with:
+        args: "Nodejs/Product/Nodejs/ProjectTemplates/**/*.js Nodejs/Product/Nodejs/ProjectTemplates/**/*.ts Nodejs/Product/Nodejs/Templates/**/*.js Nodejs/Product/Nodejs/Templates/**/*.ts"
```

Comment:
Can we lint `tsx`, `jsx`, `vue`, `html` files as well to cover everything where js/ts might be present? Is it worth it?

Reply:
Ah good catch for `tsx` and `jsx` - I'll fix that. but there isn't support for `vue` or `html` files.